src/llscan.cc

```diff
@@ -230,15 +236,53 @@ bool FindInstancesCmd::DoExecute(SBDebugger d, char** cmd,
 
   TypeRecordMap::iterator instance_it =
       llscan_->GetMapsToInstances().find(type_name);
+
   if (instance_it != llscan_->GetMapsToInstances().end()) {
     TypeRecord* t = instance_it->second;
-    for (std::set<uint64_t>::iterator it = t->GetInstances().begin();
-         it != t->GetInstances().end(); ++it) {
+
+    // Update pagination options
+    if (full_cmd != pagination_.command ||
+        inspect_options.output_limit != pagination_.output_limit) {
```

Comment:
Doesn't `full_cmd != pagination_.command` imply `inspect_options.output_limit != pagination_.output_limit`? Do we really need both?

Reply:
@mmarchini `full_cmd` is only the serch term here. Thus we need check if limit is the same.